src/Compilers/Core/Portable/InternalUtilities/CompilerFeatureRequiredAttribute.cs

```diff
@@ -5,6 +5,8 @@
 // Copied from:
 // https://github.com/dotnet/runtime/blob/fdd104ec5e1d0d2aa24a6723995a98d0124f724b/src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/CompilerFeatureRequiredAttribute.cs
 
+#if !NET6_0_OR_GREATER
```

Comment:
💡 You'll probably want a type forwarder here so the API binding surface is the same in netstandard2.0 and net6.0